### PR TITLE
LB-160: Add redis to docker setup and use redis instead of memcached

### PR DESCRIPTION
**Pull request by paramsingh**
I only found one set in the code, which I replaced with a set in redis.

### CODE REVIEW

File: webserver/external/musicbrainz.py [hunk 1]
```diff
@@ -12,7 +12,7 @@ def get_recording_by_id(mbid):
             recording = musicbrainzngs.get_recording_by_id(mbid, includes=['artists', 'releases', 'media'])['recording']
         except ResponseError as e:
             raise DataUnavailable(e)
-    cache.set(mbid, recording, time=CACHE_TIMEOUT)
+    _redis.setex(mbid, recording, CACHE_TIMEOUT)
```
Comment (alastair) [inline webserver/external/musicbrainz.py]: We should use https://github.com/metabrainz/brainzutils-python/blob/master/brainzutils/cache.py

**Review by alastair — CHANGES_REQUESTED** [review]

File: webserver/external/musicbrainz.py (hunk 1 above)
Comment (paramsingh) [reply webserver/external/musicbrainz.py]: oh, okay, will do.

**Comment by paramsingh** [comment]
Using brainzutils now :)

**Review by alastair — APPROVED** [review]

**Review by mayhem — APPROVED** [review]
Woo!

**Pull request merged**